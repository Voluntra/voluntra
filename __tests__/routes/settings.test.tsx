import renderer, { ReactTestRendererJSON } from "react-test-renderer";
import Settings from "../../src/app/settings";

describe("<Settings />", () => {
  it("has 1 child", () => {
    const tree = renderer.create(<Settings />).toJSON();
    expect((tree as ReactTestRendererJSON).children.length).toBe(1);
  });
});

it("renders correctly", () => {
  const tree = renderer.create(<Settings />).toJSON();
  expect(tree).toMatchSnapshot();
});
