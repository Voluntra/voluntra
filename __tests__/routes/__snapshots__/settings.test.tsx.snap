// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      [
        {
          "color": "#E0E0E0",
        },
      ]
    }
  >
    You're currently on the settings page!
  </Text>
</View>
`;
